.appveyor.yml

```diff
@@ -57,18 +61,49 @@ install:
       $env:path += ";C:\ponyc-windows-libs\lib"
       cd C:\projects\ponyc
       C:\premake5.exe --with-tests --to=work/vs2015 vs2015
-  - ps: |
-      $file_version = (Get-Content "C:\projects\ponyc\VERSION")
-      Update-AppveyorBuild -Version "$file_version.${env:appveyor_build_number}"
 
 after_build:
   - ps: |
-      cd C:\projects\ponyc
-      7z a -tzip "ponyc-win64-${env:appveyor_build_version}-${env:configuration}-llvm-${env:llvm}.zip" "packages\" "build\$env:configuration\ponyc.*" "build\$env:configuration\ponyrt.*"
+      md ponyc
+      md ponyc\bin
+      copy build\$env:configuration\ponyc.*  ponyc\bin
+      copy build\$env:configuration\ponyrt.* ponyc\bin
+      7z a -tzip "C:\projects\ponyc\ponyc-${env:appveyor_build_version}-win64.zip" "ponyc\bin\" "packages\"
 
 artifacts:
   - path: 'ponyc-*.zip'
 
+deploy:
+  # On branches `master` and `monthly`, deploy (and publish) artifacts
+  # to the ponyc-win projects on Bintray.
+  - provider: BinTray
+    username: pony-buildbot-2
+    api_key:
+        secure: GPMFfEx/y9lomGHzZPD+0NtI3z5g9v8nAZJ9q3O0G49aORcBBZ/VUAXwqU82d5C9
+    subject: ponylang
+    repo: ponyc-win
+    package: master
+    version: $(appveyor_build_version)
+    on:
+        branch: master
+        llvm: 3.8.0
+        configuration: Release
+    publish: true
+
+  - provider: BinTray
+    username: pony-buildbot-2
+    api_key:
+        secure: GPMFfEx/y9lomGHzZPD+0NtI3z5g9v8nAZJ9q3O0G49aORcBBZ/VUAXwqU82d5C9
+    subject: ponylang
+    repo: ponyc-win
+    package: monthly
+    version: $(appveyor_build_version)
+    on:
+        branch: monthly

```

Comment:
Perhaps here and on Bintray, `monthly` should be `release`, since we already have a `release` branch?  😅